.github/workflows/unit-tests.yml

```diff
@@ -60,11 +60,9 @@ jobs:
           - name: "Pypy"
             env_file: actions-pypy-39.yaml
             pattern: "not slow and not network and not single_cpu"
-            test_args: "--max-worker-restart 0"
```

Comment:
Is this related, or is it just not needed and we can remove the whole `test_args` thing?